tests/generator/test_post_fp.py

```diff
@@ -281,7 +281,7 @@ def setUp(self):
         post_fp(0, jdata)
         self.system_1 = dpdata.LabeledSystem("iter.000000/orig", fmt="deepmd/raw")
         self.system_2 = dpdata.LabeledSystem(
-            "iter.000000/02.fp/data.000", fmt="deepmd/raw"
+            "iter.000000/02.fp/data.000/C2H2N2", fmt="deepmd/raw"
```

Comment:
Is this ok to write multiple sys under same `sys_idx`? I remember that systems should have different suffixes, e.g. `000`, `001`

Reply:
According to my test, the modified post_fp_cp2k function will create a folder like data.xxx/sys1, data.xxx/sys2 ... in iteration xxx. Fortunately, make_train function could identify several subfolders in data.xxx.

Reply:
I think it is an expected behavior. I originally planned to modify 'make_train' in 'run.py', but it is compatible with multi-systems. Also keeping the original 'make_train' will not destroy the route with different fp methods, i.e. 'vasp' or 'QE'.

Reply:
I think it is an expected behavior. I originally planned to modify `make_train` in `run.py`, but it is compatible with multi-systems. Also keeping the original `make_train` will not destroy the route with different fp methods, i.e. `vasp` or `QE`.

Reply:
xref: #1246

Reply:
good, I got no problem